src/App.js

```diff
@@ -13,31 +20,56 @@ class App {
   }
 
   async getCarNames() {
-    try {
-      const cars = await InputView.readCarName();
+    const cars = await InputView.readCarName();
+    const validateObject = {
+      validator: () => inputCarNameValidator(cars),
+      nextStep: () => this.getTryCount(cars),
+      afterError: () => this.getCarNames(),
+    };
 
-      this.getTryCount(cars);
-    } catch (error) {
-      utils.print(error);
-      this.getCarNames();
-    }
+    ErrorHandler(validateObject);
```

Comment:
개인적으로는 굳이 이렇게까지 써야하나? 라는 생각이긴 해요.

그리고 ErrorHandler를 그대로 쓴다고 하더라도, 

```suggestion
    ErrorHandler( {
      validator: () => inputCarNameValidator(cars),
      nextStep: () => this.getTryCount(cars),
      afterError: () => this.getCarNames(),
    });
```

이렇게 바로 넘겨주는게 더 일반적인거 같아요. 
여기서 validator, nextStep, afterError는 [named parameter ](https://en.wikipedia.org/wiki/Named_parameter)와 같은 느낌으로 사용되어요.